spec/ar2dto/options_spec.rb

```diff
@@ -22,12 +32,12 @@ def self.name
           has_dto
         end
 
-        expect(klass.new.to_dto).to_not respond_to(:updated_at)
+        expect(klass.new(attributes).to_dto.updated_at).to be_nil
```

Comment:
Is this behavior desired?

Wouldn't we prefer to make it not respond to that message as it was before?

Reply:
yes maybe that's better. I wanted to define the attr_readers at the class level with all the attributes but might be a bit strange having these methods return nil. I'll try the other approach

Reply:
Agree, because if the DTO responds to these messages and returns `nil`, some people might in some cases wrongfully use the returned value as if it was the actual value.